Gemfile

```diff
@@ -1,2 +1,12 @@
 source 'https://rubygems.org'
+
 gemspec
+
+# To enable PRY, very useful for debugging
+gem 'pry'
```

Comment:
You should not keep this. It's only for development, and we neither need it at runtime nor at build time